gubernator/main_test.py

```diff
@@ -67,6 +67,9 @@ def init_build(build_dir, started=True, finished=True):
         write(build_dir + 'finished.json',
               {'result': 'SUCCESS', 'timestamp': 1406536800})
     write(build_dir + 'artifacts/junit_01.xml', JUNIT_SUITE)
+    if kubelet:
```

Comment:
Since you only use this once, move these `write` calls down to test_nodelog_kubelet.